package.json

```diff
@@ -20,7 +20,7 @@
     "analyze": "size-limit --why",
     "version": "tsdx build",
     "codegen": "yarn codegen:apollo",
-    "codegen:apollo": "apollo client:codegen --target typescript --endpoint=https://api.thegraph.com/subgraphs/name/tomprsn/yearn-vaults-v2-subgraph-mainnet --outputFlat --includes \"src/services/subgraph/apollo/**\" --excludes \"src/services/subgraph/apollo/generated\" --passthroughCustomScalars --customScalarsPrefix YearnSubgraph src/services/subgraph/apollo/generated"
+    "codegen:apollo": "apollo client:codegen --target typescript --endpoint=https://api.thegraph.com/subgraphs/name/jstashh/yearn-subgraph-three --outputFlat --includes \"src/services/subgraph/apollo/**\" --excludes \"src/services/subgraph/apollo/generated\" --passthroughCustomScalars --customScalarsPrefix YearnSubgraph src/services/subgraph/apollo/generated"
```

Comment:
We should probably switch this over to the 'official' yearn subgraph once https://github.com/yearn/yearn-vaults-v2-subgraph/pull/73 has been merged

Reply:
using the most commonly used subgraph now - https://thegraph.com/explorer/subgraph/salazarguille/yearn-vaults-v2-subgraph-mainnet